checks.d/riakcs.py

```diff
@@ -107,8 +114,16 @@ def _get_stats(self, s3, aggregation_key):
 
         return stats
 
-
-    # We need this as the riak cs stats page returns json with duplicate keys
     @classmethod
     def load_json(cls, text):
-        return json.JSONDecoder(object_pairs_hook=multidict).decode(text)
+        data = json.loads(text)
+        if "legend" in data:
+            # riak cs before v2.1 had duplicate keys
+            data = json.JSONDecoder(object_pairs_hook=multidict).decode(text)
+        return data
+
+
+STATS_METHODS = {
+    "one": "count",
+    "total": "monotonic_count",
```

Comment:
For metrics ending in `_one` or `_total` I guessed they would be better tracked as counts than a as simple gauge. It wasn't entirely clear which method was the best for each of these. More info on various types of stats here: https://github.com/basho/riak_cs/wiki/%5BRFC%5D-Riak-CS-and-Stanchion-metrics-2.1